packages/lesswrong/server/migrations/20250108T163044.remove_giving_season_flairs.ts

```diff
@@ -0,0 +1,13 @@
+import Users from "@/lib/collections/users/collection"
+import { addRemovedField, dropRemovedField } from "./meta/utils"
+import { BoolType } from "../sql/Type"
+
+export const up = async ({db}: MigrationContext) => {
+  await dropRemovedField(db, Users, "givingSeason2024DonatedFlair")
```

Comment:
We normally split "DROP" migrations into a separate PR and wait a couple of days to deploy to avoid breaking existing clients.

Reply:
Oh I forgot to submit [this comment](https://github.com/ForumMagnum/ForumMagnum/pull/10236#discussion_r1907517890) earlier. They were removed from any fragments [3 weeks ago](https://github.com/ForumMagnum/ForumMagnum/pull/10116/files) so I think it's safe now.